nilearn_sandbox/_utils/check_defaults.py

```diff
@@ -0,0 +1,167 @@
+"""Update default values in docstrings."""
```

Comment:
Could you be a bit more explicit on what is done here ? This is needed, because we will soon forget it ...